src/pkg/path/resource_path.go

```diff
@@ -31,17 +31,33 @@ func toServiceType(service string) ServiceType {
 
 var ErrorUnknownCategory = errors.New("unknown category string")
 
+// CategoryType denotes what category of data the path corresponds to. Metadata
+// categories are also included though they are only used for paths that house
+// metadata for Corso backups.
+//
+// MetadataCategory is not considered a valid category type for resource paths
+// though it can be used for metadata paths.
+//
+// The order of the enums below can be changed, but the string representation of
+// each enum must remain the same or migration code needs to be added to handle
+// changes to the string format.
 type CategoryType int
 
 //go:generate stringer -type=CategoryType -linecomment
 const (
-	UnknownCategory   CategoryType = iota
-	EmailCategory                  // email
-	ContactsCategory               // contacts
-	EventsCategory                 // events
-	FilesCategory                  // files
-	ListsCategory                  // lists
-	LibrariesCategory              // libraries
+	UnknownCategory           CategoryType = iota
+	EmailCategory                          // email
+	ContactsCategory                       // contacts
+	EventsCategory                         // events
+	FilesCategory                          // files
+	ListsCategory                          // lists
+	LibrariesCategory                      // libraries
+	EmailMetadataCategory                  // emailMetadata
+	ContactsMetadataCategory               // contactsMetadata
+	EventsMetadataCategory                 // eventsMetadata
+	FilesMetadataCategory                  // filesMetadata
+	ListsMetadataCategory                  // listsMetadata
+	LibrariesMetadataCategory              // librariesMetadata
```

Comment:
not really sure I like having to essentially duplicate categories, but I don't really know of another elegant way to handle a "metadata" category without losing information about what category stuff was for originally. We could use a flag on the Path interface, but that feels a little clunky and error prone since the stuff in the metadata directory won't be used directly during backup and restore

Reply:
Thoughts on `{tenant}/{service}/{owner}/metadata/{category}` as the path?  Treat `metadata` as its own category keyword, and re-use he existing category enums as folders?

Reply:
mmm it could work though then the code that uses the metadata path has to manually determine what category the metadata path is for because it's no longer something the Path interface can tell it. I guess it would work as long as we don't nest any folders in the `metadata/{category}` hierarchy, but having that assumption makes me uneasy

Reply:
Alterrnative:  `{tenant}/{service-metadata}/{owner}/{category}`.  Promote the "metadata" evaluation up to the service level, and again re-use the category.  Still a little duplication, but should be a smaller footprint.

Reply:
seems like a reasonable compromise